frontend/public/components/dashboard/dashboards-page/quick-start-badge.tsx

```diff
@@ -1,18 +1,19 @@
 import * as React from 'react';
 import { Label } from '@patternfly/react-core';
 import { RocketIcon } from '@patternfly/react-icons';
+import { useUserSettingsCompatibility } from '@console/shared';
 import { history } from '../../utils';
 import { useTranslation } from 'react-i18next';
 
 import './quick-start-badge.scss';
 
 const HIDE_QUICK_START_BADGE_STORAGE_KEY = 'bridge/hide-quick-start-badge';
-
+const HIDE_QUICK_START_BADGE_CONFIG_MAP = 'console.quickStartBadge';
```

Comment:
Just to confirm, Should HIDE_QUICK_START_BADGE_CONFIG_MAP and HIDE_QUICK_START_ADD_TILE_CONFIG_MAP_KEY be separate?

Reply:
What do you mean by separately hidden?

Reply:
nvm